riscv-overlay-software-standard-draft.adoc

```diff
@@ -131,39 +140,48 @@ A ‘Group’ is a collection of overlay functions and overlay data. We should u
 Example: foo(void) can be located in _Group~1~, Group~2~…, GroupN_
 
 {nbsp} +
-[[evict]]
+[[Evict]]
 ==== Evict
 
 Group eviction can be handled with similarity to HW cache concepts.
 
 1.  Eviction resolution will be at `group` granularity, meaning we can evict N groups per demand.
 2.  The search-algorithm for determining whether a group is loaded or not shall be defined at compile time.
-3.  The RT-Engine will provide a “group lock/free” API mechanism to prevent specific groups from being evicted.
+3.  The search-algorithm is open to interpretation; we recommend to have at least one, for example LRU.
+4.  The RT-Engine will provide a “group lock/free” API mechanism to prevent specific groups from being evicted.
+
+NOTE: This section is optional. But it is *_recommended_* to have it if the design requires eviction, due to heap space limitations.
 
 {nbsp} +
-[[load]]
+[[Load]]
 ==== Load
 
 The load area, “heap”, contains loaded overlay groups. It should have its own memory section definition, so that the RT-Engine and the toolchain can work on the same section.
 
 1.  The heap area should be defined pre-build.
-2.  The heap should have range limitation to be in sync with the RT-Engine and toolchain. The heap minimum size should not be smaller than the maximum pre-defined overlay group size.
+2.  The heap should have range limitation to be in sync with the RT-Engine and toolchain. The heap minimum size should be bigger or equal to the maximum pre-defined overlay group size.
+(*_heap-min-size >= max group size_*)
 3.  We can have multiple heaps to be controlled by a single/multiple RT-Engine(s).
 4.  A _Load-Function-Hook_ footnote:[Hook implementation will be the responsibility of the platform since only the platform knows how to implement them. +
 Please refer to section *_3.1.6 Platform/Framework Hooks_*] will be provided to the user for executing the load operation itself.
+5. The RT-Engine should hold all information regarding the heap. Which area is allocated/free, sizes, and address.
+6. Based on the given "heap information," the RT-Engine should provide a mechanism to lock segments in the heaps.
+7. The heap information should be acceptable by the application to get the status of the heap.
```

Comment:
`acceptable` -> `accessible`

Reply:
fixed in: #9c35a854a4a8e789db3f6078b0bc88cc47605807